src/lib/include/xtgeo/geometry.hpp

```diff
@@ -93,22 +94,32 @@ is_xy_point_in_quadrilateral(const double x,
                              const xyz::Point &p1,
                              const xyz::Point &p2,
                              const xyz::Point &p3,
-                             const xyz::Point &p4);
+                             const xyz::Point &p4,
+                             const double tolerance = numerics::TOLERANCE);
 double
```

Comment:
General notice: default values are moved from *.cpp to *.hpp which would be the right place.